pom.xml

```diff
@@ -165,7 +166,7 @@
         </repository>
 
         <!-- SmartHome p2 release repository -->
-        <repository>
+<!--        <repository>
```

Comment:
@pfink This is probably rather a temporary measure.
The problem currently is that the build resolves against some random released ESH bundles instead of using latest snapshots:
```
[INFO] Resolving dependencies of MavenProject: org.openhab.binding:org.openhab.binding.homematic:2.3.0-SNAPSHOT @ /Users/kai/Documents/dev/openhab-master/git/openhab2-addons/addons/binding/org.openhab.binding.homematic/pom.xml
[INFO] Fetching org.eclipse.smarthome.config.core_0.10.0.201803151707.jar from https://openhab.jfrog.io/openhab/eclipse-smarthome-stable/plugins/ (98.74kB)
[INFO] Fetching org.eclipse.smarthome.config.core_0.10.0.201803151707.jar from https://openhab.jfrog.io/openhab/eclipse-smarthome-stable/plugins/ (98.74kB)
[INFO] Fetching org.eclipse.smarthome.core_0.9.0.jar from http://download.eclipse.org/smarthome/updates-release/0.9.0/plugins/ (351.54kB)
[INFO] Fetching org.eclipse.smarthome.core_0.9.0.jar from http://download.eclipse.org/smarthome/updates-release/0.9.0/plugins/ (351.54kB)
[INFO] Fetching org.eclipse.smarthome.core_0.9.0.jar from http://download.eclipse.org/smarthome/updates-release/0.9.0/plugins/ (199.08kB of 351.54kB at 196.25kB/s)
[INFO] Fetching org.eclipse.smarthome.config.discovery_0.10.0.201803151707.jar from https://openhab.jfrog.io/openhab/eclipse-smarthome-stable/plugins/ (71.24kB)
[INFO] Fetching org.eclipse.smarthome.config.discovery_0.10.0.201803151707.jar from https://openhab.jfrog.io/openhab/eclipse-smarthome-stable/plugins/ (71.24kB)
^C[INFO] Fetching org.eclipse.smarthome.core.thing_0.9.0.jar from http://download.eclipse.org/smarthome/updates-release/0.9.0/plugins/ (304.47kB)
[INFO] Fetching org.eclipse.smarthome.core.thing_0.9.0.jar from http://download.eclipse.org/smarthome/updates-release/0.9.0/plugins/ (304.47kB)
```
I am not sure how we could best solve it in order to be able to build snapshots & releases through the pipeline. Can the repo declaration maybe used into profiles, so that we add this one only in a "release" profile execution?

Reply:
The `enabled` policy always refers to the [dependency version, not to the reactor artifact version](https://stackoverflow.com/questions/36177635/what-is-use-of-enabledsnapshot-releases-tag-in-maven-poms-repository-element). As the concept of snapshots does not exist in the tycho world, the tycho plugin is just ignoring this policy. If we use a profile, we should use the same naming and activation policy like here: https://github.com/openhab/openhab-distro/blob/master/pom.xml#L215

Reply:
Many thanks for the hint, @pfink, this seems to work well - I have created #3406 for it.